src/index.js

```diff
@@ -150,7 +150,7 @@ export class SlideDeck extends React.Component {
   }
 
   getMode = () => {
-    const { mode } = querystring.parse(window.location.search.replace(/^\?/, ''))
+    const { mode } = querystring.parse(window.location.search.replace(/^\?/, '')) || 'normal'
```

Comment:
Thanks! It'd be good to add some tests for this sort of thing. Based on looking at the code, does this work as expected?

Reply:
Wouldn't it be better to be explicit about it and only update `mode` when an a mode is actually defined. `mode` has a default when the state is defined.